docs/userguide/networking/work-with-networks.md

```diff
@@ -314,8 +314,8 @@ PING 172.17.0.2 (172.17.0.2): 56 data bytes
 
 ```
 
-To connect a container to a network, the container must be running. If you stop
-a container and inspect a network it belongs to, you won't see that container.
+To connect a container to a network, the container could be running or non-running. But if
+the container is non-running, inspecting the network it belongs to, you won't see that container.
 The `docker network inspect` command only shows running containers.
```

Comment:
Perhaps this should be a note (sorry, it's not really good, lacking a bit of inspiration at the moment, haha)

```Markdown
> **Note**: you can connect both running and non-running containers to a network,
> but `docker network inspect` only shows running containers.
```